src/OpenFeature.Contrib.Providers.GOFeatureFlag/GoFeatureFlagProvider.cs

```diff
@@ -53,6 +69,10 @@ private void ValidateInputOptions(GoFeatureFlagProviderOptions options)
         /// <param name="options">Options used while creating the provider</param>
         private void InitializeProvider(GoFeatureFlagProviderOptions options)
         {
+            _exporterMetadata = options.ExporterMetadata ?? new ExporterMetadata();
+            _exporterMetadata.Add("provider", ".NET");
```

Comment:
Should it be?
```suggestion
            _exporterMetadata.Add("provider", "GoFeatureFlag .NET");
```

Reply:
Since this is used by GOFF internally .NET is enough in that case.